### PR TITLE
QA-unsubscribe

### PR DESCRIPTION
**Pull request by nibivi77**
# Description

[#377 Unsubscribe Pages EN/FR](https://trello.com/c/97qjL7SL/377-377-unsubscribe-pages-en-fr)

Updated the unsubscribe translations. French and English.

## Acceptance Criteria

The text must match the Figma design.

## Test Instructions

1. Start by subscribing a email
2. once subscribed, follow the unsubscribe flow in both french and english.
3. Make sure all text matches the Figma design as well as the sizing of the text.
4. All text must also fit within the page.

## Help Requested

- I need help modifying the content sent back to the user in the unsubscribe email.
**What we have:**
image

**What we need:**
image

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated

### CODE REVIEW

**Review by var-kyle — CHANGES_REQUESTED** [review]
There is a `resend email` link that shouldn't be there on the `/thankyou` page.

image

And I noticed that the title on the unsubscribe page is `Service Canada Labs - Sign-up` and it should be `Service Canada Labs - Unsubscribe`.

Everything else looks good! I can change the email contents for you.

File: pages/thankyou.js [hunk 1]
```diff
@@ -79,7 +79,15 @@ export default function Confirmation(props) {
...
-            <ActionButton id="resend_email" text={t("resendEmail")} tertiary />
+            {referrer === "signup" ? (
+              <ActionButton
+                id="resend_email"
+                text={t("resendEmail")}
+                tertiary
+              />
+            ) : (
+              ""
+            )}
```
Comment (var-kyle) [inline pages/thankyou.js:90]: I think the resend button can just be completely removed, it doesn't look like it's used anywhere.

**Review by var-kyle — CHANGES_REQUESTED** [review]
Great work @nibivi77! Just one little thing with the resend button.

File: pages/thankyou.js (hunk 1 above)
Comment (nibivi77) [reply pages/thankyou.js:90]: Ah yes you are right. I thought we used it when subscribing. Removing...

**Review by var-kyle — APPROVED** [review]
Nice work, looks great!

File: pages/about.js [hunk 2]
```diff
@@ -78,10 +78,10 @@ export default function About(props) {
         </p>
       </section>
       <CallToAction
-        title={t("signupTitle")}
+        title={t("signupTitleCallToAction")}
         html={t("becomeAParticipantDescription")}
         href={"/signup"}
-        hrefText={t("signupTitle").slice(0, t("signupTitle").length - 1)}
+        hrefText={t("signupBtn")}
```
Comment (will0684) [inline pages/about.js:84]: I think we need this and text in the Experiments page CallToAction as well

**Review by will0684 — CHANGES_REQUESTED** [review]

File: pages/about.js (hunk 2 above)
Comment (nibivi77) [reply pages/about.js:84]: You are right! good catch

File: pages/experiments.js [hunk 3]
```diff
@@ -110,7 +110,7 @@ export default function Experiments(props) {
         title={t("signupTitle")}
```
Comment (will0684) [inline pages/experiments.js:110]: This should be signupTitleCallToAction too I believe?

**Review by will0684 — APPROVED** [review]

**Pull request merged**